flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptive/ExecutingTest.java

```diff
@@ -273,7 +274,6 @@ public void testNotifyNewResourcesAvailableBeforeCooldownIsOverScheduledStateCha
             ctx.setExpectRestarting(
                     restartingArguments -> {
                         assertThat(restartingArguments.getBackoffTime(), is(Duration.ZERO));
-                        assertThat(ctx.actionWasScheduled, is(true));
                     });
```

Comment:
```java
ctx.setExpectRestarting(
                    restartingArguments ->
                            assertThat(restartingArguments.getBackoffTime(), is(Duration.ZERO)));
```
nit: for all these removals, the brackets of the lambda expression become obsolete. But I guess, it's ok to keep the brackets for readability purposes. :thinking: